pkg/scheduler/framework/plugins/noderesources/fit.go

```diff
@@ -62,16 +66,48 @@ func (f *Fit) Name() string {
 	return FitName
 }
 
+func validateFitArgs(args config.NodeResourcesFitArgs) error {
+	var allErrs field.ErrorList
+	resPath := field.NewPath("ignoredResources")
+	for i, res := range args.IgnoredResources {
+		path := resPath.Index(i)
+		if errs := metav1validation.ValidateLabelName(res, path); len(errs) != 0 {
```

Comment:
is `ValidateLabelName` what is used to validate resource names specified in a pod or does it just happen to have similar characteristics? are all resource names valid labels?

Reply:
It is the latter. Yes, all resource names are valid labels.